boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -3,16 +3,17 @@
 Overview
 ********
 
-PocketBeagle 2 is a computational platform powered by TI AM62x SoC (there are two
-revisions, AM6232 and AM6254).
+PocketBeagle 2 is a computational platform powered by TI AM6254 SoC (rev A0,
+which has been replaced by rev A1 used AM6232).
 
 See the `PocketBeagle 2 Product Page`_ for details.
 
 Hardware
 ********
+
 PocketBeagle 2 features the TI AM62x SoC based around an Arm Cortex-A53 multicore
 cluster with an Arm Cortex-M4F microcontroller, Imagination Technologies AXE-1-16
-graphics processor (from revision A1) and TI programmable real-time unit subsystem
+graphics processor (absent in rev A0) and TI programmable real-time unit subsystem
```

Comment:
Can we just remove this note about rev A0 here? It just feels confusing. There isn't even a way to program the GPU from Zephyr.